cmd/machine-config-operator/main.go

```diff
@@ -8,10 +8,7 @@ import (
 	"k8s.io/component-base/cli"
 )
 
-const (
-	componentName      = "machine-config"
-	componentNamespace = "openshift-machine-config-operator"
-)
+const componentName = "machine-config"
```

Comment:
I think the namespace constant changes could just be their own PR (8 file changes) and merge asap.